spark/src/main/scala/geotrellis/spark/tiling/ZoomedLayoutScheme.scala

```diff
@@ -116,13 +116,15 @@ class ZoomedLayoutScheme(val crs: CRS, val tileSize: Int, resolutionThreshold: D
 
   def zoomOut(level: LayoutLevel) = {
     val layout = level.layout
+    val newZoom = level.zoom - 1
+    val newSize = math.pow(2, newZoom).toInt
```

Comment:
For zoomed layout scheme i think we can create a new, called `Square`, +that would mean a good unit test for it :)